File: 5.Jbelmud-Text/sol.go
package main

import (
	"slices"
	"strings"
)

func jbelmu(text string) string {
	words := strings.Fields(text)

	for i, w := range words {
		runes := []rune(w)
		if len(runes) <= 2 {
			continue
		}

		middle := runes[1 : len(runes)-1]
		slices.Sort(middle)
		words[i] = string(runes[0]) + string(middle) + string(runes[len(runes)-1])
	}

	return strings.Join(words, " ")
}

File: 5.Jbelmud-Text/sol.js
function jbelmu(text) {
  return text
    .split(" ")
    .map((w) =>
      w.length < 2
        ? w
        : `${w[0]}${[...w.slice(1, -1)].sort().join("")}${w[w.length - 1]}`
    )
    .join(" ");
}
